companion/src/translations/companion_fr.ts

```diff
@@ -9328,420 +9370,205 @@ Si cette option est cochée, la voie des gaz est inversée: le ralenti  est &apo
         <translation></translation>
     </message>
     <message>
-        <location filename="../simulation/simulateduiwidgetX9.cpp" line="39"/>
-        <source>PG-DN</source>
-        <translation>PAG-BAS</translation>
+        <location filename="../simulation/simulator_strings.h" line="19"/>
+        <source>&lt;font size=+3&gt;&amp;darr;&lt;/font&gt;</source>
+        <translation></translation>
     </message>
     <message>
-        <location filename="../simulation/simulateduiwidgetX9.cpp" line="39"/>
-        <source>[ PAGE ]</source>
+        <location filename="../simulation/simulator_strings.h" line="21"/>
+        <source>&lt;font size=+3&gt;&amp;#x2686;&lt;/font&gt;</source>
         <translation></translation>
     </message>
     <message>
-        <location filename="../simulation/simulateduiwidgetX9.cpp" line="43"/>
-        <source>DEL/BKSP/ESC</source>
-        <translation>EFF/RETOUR/ESC</translation>
+        <location filename="../simulation/simulator_strings.h" line="22"/>
+        <source>&lt;font size=+3&gt;&amp;#x21b6;&lt;/font&gt;</source>
+        <translation></translation>
     </message>
     <message>
-        <location filename="../simulation/simulateduiwidgetX9.cpp" line="43"/>
-        <source>[ EXIT ]</source>
-        <translation>[ SORTIR ]</translation>
+        <location filename="../simulation/simulator_strings.h" line="23"/>
+        <source>&lt;font size=+3&gt;&amp;#x21b7;&lt;/font&gt;</source>
+        <translation></translation>
     </message>
     <message>
-        <location filename="../simulation/simulateduiwidgetX9.cpp" line="62"/>
-        <source>WHEEL/PAD SCRL</source>
-        <translation>SOURIS/PAS DEFIL</translation>
+        <location filename="../simulation/simulator_strings.h" line="24"/>
+        <source>&lt;font size=+3&gt;&amp;#x21c6;&lt;/font&gt;</source>
+        <translation></translation>
     </message>
     <message>
-        <location filename="../simulation/simulateduiwidgetX9.cpp" line="47"/>
-        <source>+/UP</source>
-        <translation>+/HAUT</translation>
+        <location filename="../simulation/simulator_strings.h" line="25"/>
+        <location filename="../simulation/simulator_strings.h" line="28"/>
+        <source>&lt;font size=+3&gt;&amp;#x21d3;&lt;/font&gt;</source>
+        <translation></translation>
     </message>
     <message>
-        <location filename="../simulation/simulateduiwidgetX9.cpp" line="47"/>
-        <source>[ + ]</source>
+        <location filename="../simulation/simulator_strings.h" line="27"/>
+        <source>&lt;font size=+3&gt;&amp;#x21d1;&lt;/font&gt;</source>
         <translation></translation>
     </message>
     <message>
-        <location filename="../simulation/simulateduiwidgetX9.cpp" line="48"/>
-        <source>-/DN</source>
-        <translation>-/BAS</translation>
+        <location filename="../simulation/simulator_strings.h" line="29"/>
+        <source>&lt;font size=+3&gt;&amp;#x21d5;&lt;/font&gt;</source>
```

Comment:
Perhaps, but the HTML in this case is part of the formatting which is going to depend on the size of the characters used (eg. some use font size +3, others +2, others none, etc).  This way the whole indicator string can be replaced if necessary (eg. with an icon) w/out being tied to a particular font size (or any other attributes included in the HTML).  Granted this possibly requires a bit more "comprehension" on the part of the translator (or at least testing), but OTOH it is a lot more flexible.